### PR TITLE
Feat/upgrade to 0.27.2

### PR DESCRIPTION
**Pull request by Al-Pragliola**

### CODE REVIEW

File: test/expensive/ekscluster/furyctl_test.go [hunk 1]
```diff
@@ -148,18 +148,19 @@ var (
...
+	_ = Describe("furyctl & distro v1.25.9 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.25.9", true))
 
 	_ = Describe("furyctl & distro v1.26.0 - public minimal", CreateAndDeleteTestScenario("1.26.0", false))
 	_ = Describe("furyctl & distro v1.26.1 - public minimal", CreateAndDeleteTestScenario("1.26.1", false))
 	_ = Describe("furyctl & distro v1.26.2 - public minimal", CreateAndDeleteTestScenario("1.26.2", false))
 	_ = Describe("furyctl & distro v1.26.3 - public minimal", CreateAndDeleteTestScenario("1.26.3", false))
 	_ = Describe("furyctl & distro v1.26.4 - public minimal", CreateAndDeleteTestScenario("1.26.4", false))
 
-	_ = Describe("furyctl & distro v1.27.1 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.26.4", true))
+	_ = Describe("furyctl & distro v1.26.4 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.26.4", true))
```
Comment (nutellinoit) [inline test/expensive/ekscluster/furyctl_test.go:159]: Is this duplicated?

**Review by nutellinoit — DISMISSED** [review]
Overall LGTM with one question 👇

File: test/expensive/ekscluster/furyctl_test.go (hunk 1 above)
Comment (Al-Pragliola) [reply test/expensive/ekscluster/furyctl_test.go:159]: nope this one tests also ephemeral folder's feature

**Review by nutellinoit — APPROVED** [review]
LGTM

**Pull request merged**